entrypoint.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh -l
 
-lftp $INPUT_HOST -u $INPUT_USER,$INPUT_PASSWORD -e "set ssl:verify-certificate false; mirror --reverse --continue --dereference -x ^\.git/$ $INPUT_LOCALDIR $INPUT_REMOTEDIR; quit"
+lftp $INPUT_HOST -u $INPUT_USER,$INPUT_PASSWORD -e "set ftp:ssl-force $INPUT_FORCESSL; set ssl:verify-certificate false; mirror --reverse --continue --dereference -x ^\.git/$ $INPUT_LOCALDIR $INPUT_REMOTEDIR; quit"
```

Comment:
Thank you for your contribution! Does lftp accept `1` and `0` instead of `true` and `false`? I couldn't find anything about that in the docs. I think it would be better to transform the boolean into a string just to make sure since the [GitHub docs say](https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#operators) booleans are cast to numbers.

Reply:
You are welcome, thanks for reviewing!
You are pointing to a section called "operators" in the docs where it says "If the types do not match, GitHub coerces the type to a number.". I am not sure if this refers to input parameters of Actions. Looking at the [input metadata syntax doc](https://help.github.com/en/actions/building-actions/metadata-syntax-for-github-actions#inputs), I am under the impression that the type of input variables is always string since it is processed as an environment variable. I changed the default value to a string subsequently. Does that make sense?
Unfortunately I don't have any clue how to test a Github action properly in a workflow environment.

Reply:
Good point. Let me do some testing later this week

Reply:
Thanks for the merge and version update.
Changed my workflow file to use SSL ;)